Python/IP address info Finder/.gitignore

```diff
@@ -0,0 +1,65 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
```

Comment:
why so many files in .gitignore. You are not even using half of the things here.